.gitignore

```diff
@@ -2,3 +2,5 @@
 Gemfile.lock
 .bundle
 vendor
+.idea
```

Comment:
might be easier in your future PR to create a default `.gitignore` on your machine